day6/adventcode2015_day6.py

```diff
@@ -0,0 +1,99 @@
+# this script runs faster with pypy if large arrays are used
+
+import re
+
+# getting a list of instructions and coordinates from external file
```

Comment:
be careful about such comments in code, this one explains 2 lines, that are perfectly readable, thus being redundant comment. 

For comments that represents some tricky logic, also watch out, they tend to quickly get outdated, because no one cares about updating comments (especially you from the future) - what is even worse, compilers also don't care about comments, use them wisely/sparely